data/sql/updates/pending_db_world/rev_1676604656261986500.sql

```diff
@@ -0,0 +1,117 @@
+--
+DELETE from `creature` where `guid`=77831 AND `id1`=22060;
+INSERT INTO `creature` (`guid`, `id1`, `id2`, `id3`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `currentwaypoint`, `curhealth`, `curmana`, `MovementType`, `npcflag`, `unit_flags`, `dynamicflags`, `ScriptName`, `VerifiedBuild`) VALUES
+(77831, 22060, 0, 0, 530, 0, 0, 1, 1, 1, -2024.52, -10786.1, 81.9137, 0.664982, 3600, 0, 0, 751, 0, 2, 0, 0, 0, '', 0);
+
+UPDATE `creature_template_addon` SET `bytes1` = 0 WHERE (`entry` = 22060);
+
+-- Pathing for Entry: 22060
+SET @NPC := 77831;
+SET @path := @NPC * 10;
```

Comment:
dont use path as it is a keyword in mysql. can cause issues. Use cpath or something